services/121-service/test/payment/payment-approval.test.ts

```diff
@@ -91,4 +103,123 @@ describe('Payment approval flow', () => {
     expect(startPaymentResponseCvaManager.status).toBe(HttpStatus.FORBIDDEN);
     expect(startPaymentResponseFinanceManager.status).toBe(HttpStatus.ACCEPTED);
   });
+
+  // ##TODO: split up? Overlap with above test?
+  it('should successfully do payment that needs 2 approvals', async () => {
```

Comment:
No strong opinion from me here. A split up would be nice in the sense that each test has a single responsibility but it would slow the whole thing down